src/css/main.less

```diff
@@ -870,6 +870,10 @@ fieldset[disabled] .btn-info.active {
   }
 }
 
+.pane-header {
+  line-height: 30px;
```

Comment:
Yes, this is a really good point. Added to the list of variable candidates here:
https://github.com/mesosphere/marathon/issues/1057#issuecomment-134593084